src/payment/payment-method-action-creator.ts

```diff
@@ -53,7 +56,8 @@ export default class PaymentMethodActionCreator {
     }
 
     private _filterApplePay(methods: PaymentMethod[]): PaymentMethod[] {
-        return methods.filter(method => {
+
+        return filter(methods, method => {
```

Comment:
Note: We assume the back-end is good, so replace the vanilla `filter` with Lodash's `filter` method. Also, we have added an Array type check.